README.rst

```diff
@@ -46,7 +46,26 @@ Usage
 Example 1: How do I download the datasets?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We have `plenty of them <https://github.com/okfn-brasil/serenata-de-amor/blob/master/research/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Let's say you want your datasets at `data/`:
+We have `plenty of them <https://github.com/okfn-brasil/serenata-de-amor/blob/master/research/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Here some examples:
+
+.. code-block:: bash
+
+    # without any arguments will download pre-propre-processedd datasets and store into export/ folder
```

Comment:
There's a typo (`pre-propre-processedd`) but the main question is why changing the default to `export/`?

Usually we use Rosie's default (`/tmp/serenata-data`) or [the toolbox's default](https://github.com/okfn-brasil/serenata-toolbox/blob/master/serenata_toolbox/datasets/__init__.py#L33-L35) (`data/`).

Can we keep the toolbox's default instead of making the script behave one way and the CLI another way?

Reply:
1 It's a typo and 2 I can change the default folder.

Reply:
BTW `export is just an example of changing the already default `data` folder.